discovery-provider/alembic/versions/b3084b7bc025_filter_deletes_from_lexeme.py

```diff
@@ -0,0 +1,113 @@
+"""filter deletes from lexeme
```

Comment:
note this just copies the last migration used for lexeme and just adds the is_delete

Reply:
can we add a comment explaining this to the migration itself? just for later sanity